dev/benchmarks/platform_views_layout_hybrid_composition/android/buildscript-gradle.lockfile

```diff
@@ -1,120 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
Why was this lockfile deleted?

Reply:
The script seems to have failed on this project, I fixed it and reran.